solvers/python-DPGD.py

```diff
@@ -29,7 +27,7 @@ def run(self, callback):
         DA_inv = D @ np.linalg.pinv(self.A)
         v = np.zeros(len_y - 1)
         u = np.zeros(len_y)
-        stepsize = 1 / (np.linalg.norm(DA_inv, ord=2)**2)  # 1/ rho
+        stepsize = 1.99 / (np.linalg.norm(DA_inv, ord=2)**2)  # 1.99 / rho
```

Comment:
Does it go faster?

Reply:
![image](https://user-images.githubusercontent.com/102022158/165327588-ba8b47cc-a276-4e51-914c-f6781873194a.png)
yes, it goes a little faster.